apps/studio/components/interfaces/Organization/TeamSettings/InviteMemberButton.tsx

```diff
@@ -341,13 +341,16 @@ export const InviteMemberButton = () => {
                               <CommandEmpty_Shadcn_>No projects found</CommandEmpty_Shadcn_>
                               <CommandGroup_Shadcn_>
                                 <ScrollArea
-                                  className={(orgProjects || []).length > 7 ? 'h-[210px]' : ''}
+                                  className={cn(
+                                    (orgProjects || []).length > 7 &&
+                                      'max-h-[210px] overflow-y-auto'
+                                  )}
                                 >
                                   {orgProjects.map((project) => {
                                     return (
                                       <CommandItem_Shadcn_
                                         key={project.ref}
-                                        value={project.ref}
```

Comment:
won't this break if 2 projects have same name ?

Reply:
also this will break any form validation I guess

Reply:
> won't this break if 2 projects have same name ?

ahh good point - i could add the ref as part of the value in order to ensure each item's uniqueness, but then we won't be able to do your suggestion [as below](https://github.com/supabase/supabase/pull/28548#discussion_r1713343166)

> this will break any form validation

this one's okay - the only validation here is just to not be empty, so none broken

Reply:
pushed up a fix for the first comment - to add ref and name to the value
`onSelect` in this case will be status quo then, won't change that